### ISSUE TITLE
Could not check whether Bedrock player's name conflicts a premium Java player's name

### ISSUE THREAD

**Issue by paj1s** [games647/FastLogin#982]
### What happened?

After update to 1.19.3 i can't join because of this. (well not necessary by the update, i havent tried in a while)

### What did you expect?

To join as usual.

### Steps to reproduce

I have auto-register and auto-login to no-conflict, allowFloodgateNameConflict was to false but i also tried linked. The floodgate is set to have no prefix and my username has no special characters. It worked with this config so maybe something changed in mojangs api or maybe the account itself? I have the newest everything. 

### Plugin version

Newest

### Platform

BungeeCord

### Relevance

- [X] I tried the latest build (build refers to development builds not necessary a release version)
- [X] I checked for existing tickets -
If there are, please vote them with a thumps reaction and not create new ones

**Comment by ipozow** [comment]
You forgot to add some details. 

This happens with Geyser and Floodgate, and FastLogin Bungee [#1192](https://ci.codemc.io/job/Games647/job/FastLogin/1192/). 

It happens with Geyser Bungee [1287](https://ci.opencollab.dev/job/GeyserMC/job/Geyser/job/master/1287/), [#1286](https://ci.opencollab.dev/job/GeyserMC/job/Geyser/job/master/1286/), [#1285](https://ci.opencollab.dev/job/GeyserMC/job/Geyser/job/master/1285/), [#1284](https://ci.opencollab.dev/job/GeyserMC/job/Geyser/job/master/1284/), [#1272](https://ci.opencollab.dev/job/GeyserMC/job/Geyser/job/master/1272/), [#1271](https://ci.opencollab.dev/job/GeyserMC/job/Geyser/job/master/1271/), and [#1270](https://ci.opencollab.dev/job/GeyserMC/job/Geyser/job/master/1270/). Basically any Geyser BungeeCord version. 

Floodgate versions: [#74](https://ci.opencollab.dev/job/GeyserMC/job/Floodgate/job/master/74/) and [#73](https://ci.opencollab.dev/job/GeyserMC/job/Floodgate/job/master/73/).

**Comment by paj1s** [comment]
And what is the cause of it?

**Comment by rapofil** [comment]
the same problem

**Comment by rapofil** [comment]
without fastlogin bedrock players will join

**Comment by Teleriann** [comment]
Yes, same problem here, i have not changed any settings at all, players get this screen: 
![image](https://user-images.githubusercontent.com/71602218/216739212-271999c2-3fc9-4f17-933b-749a4c9a0e05.png)

and i can see this in my console:
![image](https://user-images.githubusercontent.com/71602218/216739239-a530af9a-7463-4d86-bfc5-15aaefb8c482.png)

tried to update fastlogin, geyser, waterfall/bungeecord, floodgate, nothing works

**Comment by Scoooolzs** [comment]
i have this error too, but i'm not using bungee, only spigot

**Comment by Smart123s** [comment]
Is the issue still present? Everything works fine on my end today.

**Comment by ACluco1** [comment]
is this going to be fixed someday?

**Comment by ACluco1** [comment]
Im just using Geyser, FastLogin, Floodgate, ProtocolLib, and Purpur (which is a paper fork), just single server without bungee, im getting messages that fastlogin could not check whether Bedrock Player name conflicts a premium Java player's name. When i disable fastlogin, bedrock players can join.

**Comment by Smart123s** [comment]
Does chaging `allowFloodgateNameConflict` to `true` help?

**Comment by ACluco1** [comment]
> Does changing`allowFloodgateNameConflict` to `true` help?

I somehow cannot find it in the config

**Comment by Smart123s** [comment]
Here's the latest [config.yml](https://github.com/games647/FastLogin/blob/main/core/src/main/resources/config.yml), try copying the missing values from it.
Also, is your bedrock account linked to a java account?

**Comment by ACluco1** [comment]
> Here's the latest [config.yml](https://github.com/games647/FastLogin/blob/main/core/src/main/resources/config.yml), try copying the missing values from it. Also, is your bedrock account linked to a java account?

Yes, now i am able to join with Bedrock, when `allowFloodgateNameConflict` is set to `true`.

**Comment by Smart123s** [comment]
> > Here's the latest [config.yml](https://github.com/games647/FastLogin/blob/main/core/src/main/resources/config.yml), try copying the missing values from it. Also, is your bedrock account linked to a java account?
> 
> Yes, now i am able to join with Bedrock, when `allowFloodgateNameConflict` is set to `true`.

Thanks for your help. Just don't forget that with that option enabled, a bedrock player with the same name as your java account's name will be able to auto login.

**Comment by ACluco1** [comment]
> > > Here's the latest [config.yml](https://github.com/games647/FastLogin/blob/main/core/src/main/resources/config.yml), try copying the missing values from it. Also, is your bedrock account linked to a java account?
> > 
> > 
> > Yes, now i am able to join with Bedrock, when `allowFloodgateNameConflict` is set to `true`.
> 
> Thanks for your help. Just don't forget that with that option enabled, a bedrock player with the same name as your java account's name will be able to auto login.

Will they be able to autologin or just join the server? There is Authme installed so it would be better if it will be asking for login with password for bedrock players, or they will just autologin?
